libraries/interop-ribs/src/main/kotlin/com/bumble/appyx/interop/ribs/InteropActivity.kt

```diff
@@ -4,25 +4,26 @@ import android.content.Intent
 import android.os.Bundle
 import com.badoo.ribs.android.RibActivity
 import com.bumble.appyx.core.integrationpoint.ActivityIntegrationPoint
+import com.bumble.appyx.core.integrationpoint.IntegrationPointProvider
 
-abstract class InteropActivity : RibActivity() {
+abstract class InteropActivity : RibActivity(), IntegrationPointProvider {
 
-    private lateinit var integrationPointAppyx: ActivityIntegrationPoint
+    override lateinit var appyxIntegrationPoint: ActivityIntegrationPoint
```

Comment:
conflicts with RIBs integrationPointAppyx property. I can either it leave as `appyxIntegrationPoint` or change it to `integrationPoint` but we'll have to rename RIBs' `appyxIntegrationPoint` then